### PR TITLE
clean up several issues and clarify a few sections

### PR DESCRIPTION
**Pull request by mprorock**
Added another example, along side cleaning up a few issues noted in the spec.

Additionally reorganized a few text locations for clarity and to avoid repetition

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -161,15 +206,18 @@ <h1>
...
+        
         <p>
 The target system of the Web DID method is the web host that the domain name
 described by the DID resolves to when queried through the Domain Name System
 (DNS).
         </p>
+        
+        <p>
+          Unlike [[?DID-DNS]], where the DID is identified as part of a DNS URI 
+          Resource Record, did:web permits both discover of and delivery of a
```
Comment (mprorock) [inline index.html]:
```suggestion
          Resource Record, did:web permits both discovery of and delivery of a 
```

File: index.html [hunk 2]
```diff
@@ -222,6 +270,12 @@ <h2>
 DID method operations
         </h2>
         <section>
+          <p>
+There is no HTTP API specified for did:web method operations process leaving programmatic
```
Comment (mprorock) [inline index.html]:
```suggestion
There is intentionally no HTTP API specified for did:web method operations leaving programmatic
```

File: index.html [hunk 3]
```diff
@@ -161,15 +206,18 @@ <h1>
...
+        
         <p>
 The target system of the Web DID method is the web host that the domain name
 described by the DID resolves to when queried through the Domain Name System
 (DNS).
         </p>
+        
+        <p>
+          Unlike [[?DID-DNS]], where the DID is identified as part of a DNS URI 
+          Resource Record, did:web permits both discovery of and delivery of a
```
Comment (OR13) [inline index.html]: I think we don't need to comment on DID DNS.

**Review by OR13 — APPROVED** [review]

File: index.html (hunk 3 above)
Comment (mprorock) [reply index.html]: agreed - not necessary,  but not harmful (and potentially helpful to folks that are net new to dids [pun intended]) and is a question i have heard asked by multiple folks many times

Comment (gribneau) [reply index.html]: +1

The method is different entirely from `did:dns`.

**Review by gribneau — CHANGES_REQUESTED** [review]
The additional example appears to be `did:key`:

```
  "id": "did:key:z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn",
```

**Comment by mprorock** [comment]
> The additional example appears to be `did:key`:
> 
> ```
>   "id": "did:key:z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn",
> ```

yep - this is pretty common in practice, especially in dev environments - we could change to `did:example` @OR13 thoughts?

**Comment by OR13** [comment]
for did web, I would use `did:web` examples ONLY... no other DID Methods should be referenced imo... unless there is a really good reason.

File: index.html [hunk 4]
```diff
@@ -149,6 +146,54 @@ <h2>
...
+        <pre class="example" title="Example did:web DID document using JWS2020">
+{
+  "@context": [
+    "https://www.w3.org/ns/did/v1",
+    "https://w3id.org/security/suites/jws-2020/v1"
+  ],
+  "id": "did:key:z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn",
+  "verificationMethod": [
+    {
+      "id": "did:key:z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn#z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn",
```
Comment (OR13) [inline index.html]:
```suggestion
      "id": "did:web:example.gov#z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn",
```

**Review by OR13 — CHANGES_REQUESTED** [review]
remove references to did:key

File: index.html (hunk 4 above)
Comment (OR13) [reply index.html]: this has to match the did.... so maybe don't take my suggestion.

**Comment by gribneau** [comment]
If we're using this additional example to provide clarity to the reader, perhaps explaining that `id` indicates the subject of this document, and that the subject could use a different DID method would have value.

The term `id` is not immediately intuitive as an external reference, but an explanation could highlight interoperability between methods.

**Comment by OR13** [comment]
`id` is the only normative requirement for a did document... and its defined by did core: 

https://www.w3.org/TR/did-core/#did-subject

`id` for a did web, should be a DID, that is of the `did:web:` format.

**Comment by gribneau** [comment]
> `id` is the only normative requirement for a did document... and its defined by did core:
> 
> https://www.w3.org/TR/did-core/#did-subject
>
> `id` for a did web, should be a DID, that is of the `did:web:` format.

Yes, I realize that the DID subject (`id`) is typically the same as the DID example documents, but is that a requirement?

https://www.w3.org/TR/did-core/#also-known-as

In the event that a single subject uses more than one DID, the subject (`id`) in at least one of the associated documents will match the subject, rather than the DID being resolved to that document.

In this scenario, is there any reason that the DID methods used by the subject and multiple identifiers it uses must match?

File: index.html [hunk 5]
```diff
@@ -149,6 +146,54 @@ <h2>
...
 }
         </pre>
+        
+        <pre class="example" title="Example did:web DID document using JWS2020">
+{
+  "@context": [
+    "https://www.w3.org/ns/did/v1",
+    "https://w3id.org/security/suites/jws-2020/v1"
+  ],
+  "id": "did:key:z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn",
```
Comment (mprorock) [inline index.html]:
```suggestion
  "id": "did:web:example.com#key-0",
```

File: index.html (hunk 4 above)
Comment (mprorock) [inline index.html]:
```suggestion
      "id": "did:web:example.com#key-0",
```

Comment (mprorock) [reply index.html]: yeah - this is such a common use case wasn't sure how we wanted to handle

File: index.html [hunk 6]
```diff
@@ -149,6 +146,54 @@ <h2>
...
+  "verificationMethod": [
+    {
+      "id": "did:key:z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn#z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn",
+      "type": "JsonWebKey2020",
+      "controller": "did:key:z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn",
```
Comment (mprorock) [inline index.html:161]:
```suggestion
      "controller": "did:web:example.com#owner",
```

File: index.html [hunk 7]
```diff
@@ -149,6 +146,54 @@ <h2>
...
+      "publicKeyJwk": {
+        "kty": "OKP",
+        "crv": "Ed25519",
+        "x": "0-e2i2_Ua1S5HbTYnVB0lj2Z2ytXu2-tYmDFf8f5NjU"
+      }
+    },
+    {
+      "id": "did:key:z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn#z6LSt8Cke1XG6vthTpdek9xx6a5NKz8gEuYPQHJPhRjfREAC",
```
Comment (mprorock) [inline index.html]:
```suggestion
      "id": "did:web:example.com#key-0",
```

File: index.html [hunk 8]
```diff
@@ -149,6 +146,54 @@ <h2>
...
+      "type": "JsonWebKey2020",
+      "controller": "did:key:z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn",
```
Comment (mprorock) [inline index.html:171]:
```suggestion
      "controller": "did:web:example.com#owner-2",
```

**Comment by mprorock** [comment]
> remove references to did:key

@OR13 please take a look at the suggestions i just dropped in to see if those make sense to you as an adjustment

File: index.html (hunk 5 above)
Comment (OR13) [reply index.html]: ^ nope... `id` MUST be a DID, NOT a DID URL!

File: index.html (hunk 4 above)
Comment (OR13) [reply index.html]: this is ok, you can use a less confusing fragment if you wish.

Comment (OR13) [reply index.html]: weird choice... don't like...

File: index.html (hunk 7 above)
Comment (OR13) [reply index.html]: not sure why you keep using `#owner`... the spec usually uses `#key-0` when in doubt.

File: index.html (hunk 3 above)
Comment (OR13) [reply index.html]: IMO, its harmful to connect readers to DID DNS... i'm a -1 to including references to it.

**Review by OR13 — CHANGES_REQUESTED** [review]
Remove reference to DID DNS, fix `id` DID and DID URL issues.

File: index.html (hunk 3 above)
Comment (gribneau) [reply index.html]: -1 to including references to `did:dns`, +1 to removing.

File: index.html [hunk 9]
```diff
@@ -142,380 +147,543 @@ <h2>
...
+        "kty": "EC",
+        "crv": "P-256",
+        "x": "38M1FDts7Oea7urmseiugGW7tWc3mLpJh6rKe7xINZ8",
+        "y": "nDQW6XZ7b_u2Sy9slofYLlG03sOEoug3I0aAPQ0exs4"
+      }
+    },
+  ],
+  "authentication": [
+    "did:key:z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn#z6MktiSzqF9kqwdU8VkdBKx56EYzXfpgnNPUAGznpicNiWfn",
+    "did:example:123#_TKzHv2jFIyvdTGF1Dsgwngfdg3SH6TpDv0Ta1aOEkw"
```
Comment (mprorock) [inline index.html]:
```suggestion
    "did:web:example.com#key-0",
    "did:web:example.com#key-1"
```

File: index.html [hunk 10]
```diff
@@ -142,380 +147,543 @@ <h2>
...
-The target system of the Web DID method is the web host that the domain name
-described by the DID resolves to when queried through the Domain Name System
-(DNS).
-        </p>
-      </section>
+      <p>
+        Unlike [[?DID-DNS]], where the DID is identified as part of a DNS URI
+        Resource Record, did:web permits both discovery of and delivery of a
+        DID document over HTTPS.
+      </p>
```
Comment (mprorock) [inline index.html]:
```suggestion
```

File: index.html (hunk 5 above)
Comment (OR13) [reply index.html]: This should be ` "id": "did:web:example.com",`

File: index.html (hunk 6 above)
Comment (OR13) [inline index.html]: controller MUST be a DID.
controller MUST NOT be a DID URL.

File: index.html (hunk 8 above)
Comment (OR13) [inline index.html]: same comment, controller must be a DID, not a DID URL

**Review by OR13 — CHANGES_REQUESTED** [review]
Make 

didDocument.id a DID (did:web:example.com)
didDocument.verificationMethod[0].controller a DID  (did:web:example.com)

File: index.html (hunk 8 above)
Comment (mprorock) [reply index.html]: ok - trying to figure out what we want here - adjust the id to did:web, and keep controller as the did:key?

Comment (OR13) [reply index.html]: controller must be did web.... in general.. mixing controllers for verification methods that don't match the did document subject is legal... but not something that is interoperable.... so don't use examples that do this.

Comment (mprorock) [reply index.html]: excellent - thanks

**Comment by mprorock** [comment]
> Make
> 
> didDocument.id a DID (did:web:example.com) didDocument.verificationMethod[0].controller a DID (did:web:example.com)

believe this is resolved now @OR13

**Review by OR13 — APPROVED** [review]

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

**Review by OR13 — APPROVED** [review]

**Pull request merged**